project/website.md

```diff
@@ -0,0 +1,35 @@
+# Pacifica Main Website
+
+This is a space to develop some of the text required for the website.
+
+## Home Page
+
+This section is for the home page text, maybe links to images.
+
+### Top Page
+
+Collaborate. Share. Integrate. Build. 
+Pacifica supports the scientific data lifecycle.
+
+### Data Management
+
+Scientific Data Management systems for research and development
+institutions require many solutions to already complex problems.
+Pacifica has solutions to manage data between projects that are
+completing and collaborative. Pacifica can accelerate the scientific
+process by making search and discovery tracta
+
```

Comment:
Scientific data management systems for research and development institutions need to provide uncomplicated solutions to meet often complicated problems. Pacifica offers institutions a platform to collaboratively manage data between projects. 

... looks like unfinished section. LMK when to take another look at it.

Reply:
Yup I screwed up on this... didn't complete the section good catch.

Reply:
Done.